.markdownlint/style.rb

```diff
@@ -0,0 +1,31 @@
+rule 'MD002'
+rule 'MD003'
+rule 'MD004', :style => :dash
+rule 'MD005'
+rule 'MD006'
+rule 'MD007'
+rule 'MD009'
+rule 'MD010'
+rule 'MD011'
+rule 'MD012'
+rule 'MD013', :line_length => 360
+rule 'MD014'
+rule 'MD018'
+rule 'MD019'
+rule 'MD020'
+rule 'MD021'
+rule 'MD022'
+rule 'MD023'
+rule 'MD024'
+rule 'MD026'
+rule 'MD027'
+rule 'MD028'
+rule 'MD029'
+rule 'MD030'
+rule 'MD031'
+rule 'MD032'
+rule 'MD035'
+rule 'MD037'
+rule 'MD038'
+rule 'MD039'
+rule 'MD046'
```

Comment:
Let's add names of the rules or link to them in a comment in the header or both :)

https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md

Reply:
Done, added the link to these docs